teuthology/ceph.conf.template

```diff
@@ -1,3 +1,13 @@
+# XXX
+#
+#   DO NOT MODIFY THIS FILE
+#
+# This file is a legacy ceph.conf template used only when testing older
+# releases of Ceph (pre-Nautilus). The new template exists in ceph.git at
+# qa/tasks/ceph.conf.template
+#
+# XXX
```

Comment:
what is that XXX?

Reply:
Many text editors show "XXX" in red highlighting to draw attention.

Reply:
oh... good to know thanks